lib/langchain/llm/aws_bedrock.rb

```diff
@@ -105,6 +106,26 @@ def complete(prompt:, **params)
       parse_response response
     end
 
+    def chat(messages: [], system: nil, **params)
```

Comment:
Instead of this `**params`, I'd like to actually be super explicit (similar to https://github.com/andreibondarev/langchainrb/blob/main/lib/langchain/llm/openai.rb#L105-L125). The reason for that is because as a developer, when you look at `**params`, you have no clue what's actually expected. For the interfaces that you control, such as this one, it makes sense to enumerate all of the different options.